alot/commands/__init__.py

```diff
@@ -34,6 +67,7 @@ class CommandCanceled(Exception):
     """
     pass
 
+
```

Comment:
That's how I usually follow PEP8 on the top level. Two blank lines between "stuff". In this case, the end of the `CommandCanceled` class and the top level `COMMANDS` dictionary initialization. There already was two blank lines after the `COMMANDS` dictionary initialization before `def lookup_command(cmdname, mode)`, so I added an extra to the spacing above.